src/test/java/io/jenkins/pluginhealth/scoring/PluginServiceTest.java

```diff
@@ -0,0 +1,63 @@
+package io.jenkins.pluginhealth.scoring;
+
+import static org.junit.Assert.assertEquals;
+
+import io.jenkins.pluginhealth.scoring.model.Plugin;
+import io.jenkins.pluginhealth.scoring.repository.PluginRepository;
+import io.jenkins.pluginhealth.scoring.service.PluginService;
+
+import org.jetbrains.annotations.NotNull;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.util.TestPropertyValues;
+import org.springframework.context.ApplicationContextInitializer;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.test.context.ContextConfiguration;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+@ContextConfiguration(initializers = PluginServiceTest.DockerPostgresDatasourceInitializer.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@Testcontainers
+@RunWith(MockitoJUnitRunner.class)
+public class PluginServiceTest {
+    private final PluginRepository pluginRepository;
+    private final PluginService pluginService;
+
+    Plugin plugin = new Plugin("myPlugin", "https://github.com/jenkinsci/my-plugin", null);
```

Comment:
You shouldn't create the plugin as a class field. We could want to test that we can create multiple plugins. 
Those kind of object should be limited to a test method.